doc/source/index.rst

```diff
@@ -117,14 +117,18 @@ Ray provides Python, Java, and *EXPERIMENTAL* C++ API. And Ray uses Tasks (funct
     | The C++ Ray API is currently experimental with limited support. You can track its development `here <https://github.com/ray-project/ray/milestone/17>`__ and report issues on GitHub.
     | Run the following commands to get started:
     | - Build ray from source with *bazel* as shown `here <https://docs.ray.io/en/master/development.html#building-ray-full>`__.
-    | - Run `"cd ray/cpp"`.
-    | - Run `"cp dev_BUILD.bazel BUILD.bazel"`.
-    | - Modify `src/ray/example/example.cc`.
+    | - Modify `cpp/example/example.cc`.
+    | - Run `"bazel build //cpp:example"`.
+    | The first way, you can run the example directly with a dynamic library path. It will start a Ray cluster automatically.
+    | - Run `"ray stop"`.
+    | - Run `"./bazel-bin/cpp/example/example --dynamic-library-path=bazel-bin/cpp/example/example.so"`
+    | The second way, you can connect to a existing Ray cluster with an additional redis address.
+    | - Run `"ray stop"`.
+    | - Run `"ray start --head --port 6378 --redis-password 5241590000000000 --node-manager-port 62665"`.
+    | - Run `"./bazel-bin/cpp/example/example --dynamic-library-path=bazel-bin/cpp/example/example.so --redis-address=127.0.0.1:6378"`.
     | - Run `"ray stop"`.
```

Comment:
```suggestion
    | Option 2: connect to an existing Ray cluster with a known redis address (e.g. `127.0.0.1:6379`).
    | - Run `"./bazel-bin/cpp/example/example --dynamic-library-path=bazel-bin/cpp/example/example.so --redis-address=127.0.0.1:6379"`.
```
Shouldn't we be able to directly connect to an existing cluster? Calling `ray stop` will stop all the clusters. What we want is to directly call:
`"./bazel-bin/cpp/example/example --dynamic-library-path=bazel-bin/cpp/example/example.so --redis-address=127.0.0.1:6378"`

We should not call `ray stop` at the end because it will destroy the cluster.

What do you think?

Reply:
You should guarantee the user has started a Ray cluster. I have add a step:
| - Run `"ray start --head --port 6378 --redis-password 5241590000000000 --node-manager-port 62665"`.
The `"ray stop"` at the beginning is to make sure `ray start` is run only once.
The `"ray stop"` at the end can be deleted. I just want to give back a clean environment with no ray processes.